labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -19,7 +19,13 @@ object ListManipulationExercise02 {
    * of the two list
    */
   def sumOfTwo(l1: List[Int], l2: List[Int]): List[Int] = {
-    error("fix me")
+    if (l1.isEmpty) {
+      l2
+    } else if (l2.isEmpty) {
+      l1
+    } else {
+      l1.zip(l2).map(x => x._1 + x._2)
```

Comment:
This seems nice, and it's very different from how I did it (recursively). I'm not sure if it will do what it's supposed to on something like a list of length 1 and a list of length 2 (which again the unit tests don't test). I'm not certain what the behavior of zip is on lists of different lengths - I think you might be able to get a nice version of this by padding the shorter list with zeros and then calling zip.